acasclient/lsthing.py

```diff
@@ -850,7 +849,18 @@ def as_camel_dict(self):
         snake_case_dict = self.as_dict()
         camel_dict = convert_json(snake_case_dict, underscore_to_camel)
         return camel_dict
+    
+    def __deepcopy__(self, memo):
+        # Create a deep copy of the instance first
+        copied_obj = copy.copy(self)  # Shallow copy of self
+        # Then deep copy the attributes individually
+        copied_obj.__dict__ = copy.deepcopy(self.__dict__, memo)
+        # Set the 'id' attribute to None
+        copied_obj.id = None
```

Comment:
While it makes sense to put this nulling out into this function, someone coming across this function wouldn't really have reason to expect it will clear out these specific keys. I think this function should have a docstring explaining that it has this special behavior.

Reply:
Thanks, yeah, just added a clearer doc string on this.